#! /bin/sh /usr/share/dpatch/dpatch-run
# Description: mysqlhotcopy uses libdbd-mysql-perl built against libmysqlclient16 from mysql5.1 but it loads the one from mysql7.1 which has a binary incompatibility

@DPATCH@
diff -urNad mysql-cluster-gpl-7.1.15a.orig/mysql-test/t/disabled.def mysql-cluster-gpl-7.1.15a/mysql-test/t/disabled.def
--- mysql-cluster-gpl-7.1.15a.orig/mysql-test/t/disabled.def	2011-10-10 19:02:59.000000000 +0100
+++ mysql-cluster-gpl-7.1.15a/mysql-test/t/disabled.def	2011-10-12 09:46:15.000000000 +0100
@@ -13,3 +13,7 @@
 query_cache_28249        : Bug#43861 2009-03-25 main.query_cache_28249 fails sporadically
 log_tables-big           : Bug#48646 2010-11-15 mattiasj report already exists
 read_many_rows_innodb    : Bug#37635 2010-11-15 mattiasj report already exists
+
+mysqlhotcopy_archive     : libmysqlclient16 binary incompatibility (libdbd-mysql-perl uses system mysql 5.1 version but tries loading cluster 7.1 version)
+mysqlhotcopy_myisam      : libmysqlclient16 binary incompatibility (libdbd-mysql-perl uses system mysql 5.1 version but tries loading cluster 7.1 version)
+
